vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -95,12 +95,15 @@ class FlyoutShadowNode : public ShadowNodeBase {
   float m_verticalOffset = 0;
   bool m_isFlyoutShowOptionsSupported = false;
   winrt::FlyoutShowOptions m_showOptions = nullptr;
+  static std::int32_t s_cOpenFlyouts;
```

Comment:
>static [](start = 2, length = 6)

I'd recommend making this a thread_local static to store the counter in thread-local-storage, as it's theoretically possible to create more than one UI thread with this framework.

Reply:
Ok. ..update PR.

Reply:
looks good thanks!

---
In reply to: [300805965](https://github.com/microsoft/react-native-windows/pull/2733#discussion_r300805965) [](ancestors = 300805965)